docroot/profiles/custom/cgov_site/modules/custom/cgov_infographic/src/Controller/CGovInfographicController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\cgov_infographic\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Controller routines for cgov_infographic routes.
+ */
+class CGovInfographicController extends ControllerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getModuleName() {
+    return 'cgov_infographic';
+  }
+
+  /**
+   * Return an infographic's long description in an HTTP Response.
+   *
+   * @param int $node
```

Comment:
This either should be `$media` or `$entity` calling a media item as a node hurts my head greatly. I am sure @bkline would agree on this one. I would say media as it is only for media.

Additionally, if it is an id, then $node DEFINITELY should not be the variable name. I thought this was the media entity until I got further down in the code.

Reply:
Changed to $media.